kfdefs/base/trino/kfdef.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  annotations:
+    kfctl.kubeflow.io/force-delete: "false"
+  name: opendatahub
+spec:
+  applications:
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: odh-common
+      name: odh-common
+    - kustomizeConfig:
+        parameters:
+        - name: s3_endpoint_url
+          value: s3.odh.com
```

Comment:
@HumairAK should this be something different?

Reply:
yeah I've sent the url in slack to @rimolive

Reply:
@anishasthana @HumairAK this should be updated now and ready to merge :+1: